packages/devtools_app/lib/src/http/_http_date.dart

```diff
@@ -378,9 +386,9 @@ class HttpDate {
     }
 
     int year = toInt(yearStr);
-    if (year >= 70 && year <= 99)
+    if (year >= 70 && year <= 99) {
       year += 1900;
-    else if (year >= 0 && year <= 69) year += 2000;
+    } else if (year >= 0 && year <= 69) year += 2000;
```

Comment:
DBC: it's interesting that no lints / style guide violations appear w/ the multi-line `if` body pared with an in-line `else if` body

Reply:
good catch!  

https://github.com/dart-lang/linter/issues/2416